.github/workflows/pr.yml

```diff
@@ -0,0 +1,61 @@
+name: UI5-Language-Assistant
+
+on:
+  
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+    env:
+      HUSKY_SKIP: "true"
+      # NODE_OPTIONS: "--openssl-legacy-provider"
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x, 18.x]
+        os: [macos-latest, ubuntu-latest, windows-latest]
```

Comment:
I think this matrix is too large considering the free tier of github actions has
strict limits on max parallel jobs and this limits are counted towards the whole SAP org afaik

- https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration